backend/Containerfile

```diff
@@ -0,0 +1,16 @@
+FROM  registry.access.redhat.com/ubi8/python-38 as base
+USER 0
+RUN mkdir /ibutsu_venv
+RUN chown -R 1001:0 /ibutsu_venv /srv
+USER 1001
+WORKDIR  /srv
+RUN python -m venv /ibutsu_venv && \
+     /ibutsu_venv/bin/pip install -U pip setuptools wheel
+
+
+FROM base as installed
+
+COPY .  /srv/
+RUN /ibutsu_venv/bin/pip install -e .
+ENTRYPOINT [ "/bin/bash", "/srv/docker/entrypoint.sh" ]
```

Comment:
I'm wondering why don't you use `dumb-init` here?

Reply:
the entrypoint here is a shell that runs actions and then starts subprocesses, as such it correctly handles the signals

if i where to exec into something, then dumb init would be needed

Reply:
good catch, will prepare a followup